framework/test/mocha/integration/blocks/verify/verify.js

```diff
@@ -954,15 +955,84 @@ describe('blocks/verify', () => {
 	});
 
 	describe('processBlock for invalid block {broadcast: true, saveBlock: true}', () => {
-		beforeEach(done => {
-			blocksVerify.processBlock(block1, true, true, done);
+		it('should fail when processing block 1 multiple times', done => {
+			blocksVerify.processBlock(block1, true, true, () => {
+				blocksVerify.processBlock(block1, true, true, err => {
+					expect(err).to.equal('Invalid block timestamp');
+					done();
+				});
+			});
 		});
 
-		it('should fail when processing block 1 multiple times', done => {
-			blocksVerify.processBlock(block1, true, true, err => {
-				expect(err).to.equal('Invalid block timestamp');
-				done();
+		it('should fail when block contains transactions in order to make total spending more than balance', async () => {
+			const promisifyProcessBlock = util.promisify(blocksVerify.processBlock);
+			const currentSlot = slots.getSlotNumber();
+
+			// Create a random account
+			const account = random.account();
+
+			// Transfer 1 Lisk to that account...
+			const block = createBlock(
+				blocks,
+				blockLogic,
+				await getValidKeypairForSlot(library, currentSlot - 1),
+				slots.getSlotTime(currentSlot - 1),
+				[
+					transfer({
+						amount: new Bignum(NORMALIZER).multipliedBy(1).toString(),
+						recipientId: account.address,
+						passphrase: accountFixtures.genesis.passphrase,
+					}),
+				],
+				genesisBlock
+			);
+			// Process the block so account have 1 LSK
+			await promisifyProcessBlock(block, true, true);
+
+			// Transfer 0.5 LSK to a random account - The account balance would be 0.4
+			const trs_05_1 = transfer({
```

Comment:
How about making it more simple
```
const transactions = [0.5, 0.5, 0.3].map(amount => transfer({
				amount: new Bignum(NORMALIZER).multipliedBy(amount).toString(),
				recipientId: random.account().address,
				passphrase: account.passphrase,
			}));
```

Reply:
Only works for `trs_05_01` and `trs_05_02`